ibm/service/vpc/resource_ibm_is_network_acl_rule.go

```diff
@@ -607,13 +607,15 @@ func nwaclRuleUpdate(d *schema.ResourceData, meta interface{}, id, nwACLId strin
 			updateNetworkACLOptionsPatchModel.Action = &action
 		}
 	}
-
+	aclRuleBeforeNull := false
```

Comment:
Please add acceptance test case.. 
Thank you

Reply:
added